### ISSUE TITLE
no-unused-variable not reporting unused imports

### ISSUE THREAD

**Issue by HamletDRC** [palantir/tslint#1359]
### Bug Report

- __TSLint version__: `3.11.0`
- __TypeScript version__: `1.8.10`
- __Running TSLint via__: (please choose one) grunt-tslint 

#### TypeScript code being linted

```ts
import ActionContext = require('post-interactions/models/ActionContext');
import ActionErrorHandler = require('post-interactions/models/action-error-handling/handler/ActionErrorHandler');
import CacheablePostsFetcher = require('post-interactions/models/CacheablePostsFetcher');
import FetcherErrorHandler = require('post-interactions/models/action-error-handling/handler/FetcherErrorHandler');
import PostActionErrorReports = require('post-interactions/models/action-error-handling/PostActionErrorReports');

/**
 * Handle errors for the post details view.
 */
class PostDetailsErrorHandler {

    public canHandle(): boolean {
        return false;
    }

    public handle(): JQueryPromise<void> {
        return null;
    }
}

export = PostDetailsErrorHandler;
```

with `tslint.json` configuration:

```json
    "no-unused-variable": true,
    "no-unused-imports": true,
```

#### Actual behavior

the no-unused-imports correctly reports the 5 unused imports. 
the no-unused-variable rule reports 0 unused imports

#### Expected behavior

There should be 5 unused variable violations

**Comment by HamletDRC** [comment]
This turned out to be a problem on our side in rules configuration. Some rules were being turned off.
